### ISSUE TITLE
epubchecker3.0-RC-1 have following bug & need to be fixed

### ISSUE THREAD

**Issue by rdeltour** [w3c/epubcheck#216]
_From [jaiar...@gmail.com](https://code.google.com/u/104231621411000324062/) on October 27, 2012 08:12:57_

PROBLEM: EPC 3.0 generating different reports for defaul/epub/exp modes & opf/xhtml modes. METHOD: executing epubcheck-3.0-RC-1 with XHTML/opf modes 
throws different errors rather executing in default/Epub/exp modes RESULT: 1. Errors reported in opf mode missed in default/Epub/exp modes!
2. Unnecessary schema errors are thrown in XHTML modes... EXPECTED: whatever the mode the error/warning list should be same as per the mode PLATFORM: epubchecker 3.0-RC-1/windows-xp ADDITIONAL INFO: following errors are thrown in opf mode missed in default/epub/exp modes

```
D:\arul\EPC\epubcheck-3.0-RC-1\epubcheck-3.0-RC-1>java -jar epubcheck-3.0-RC-1.j
ar D:\Temp\Test_9781118388341\OEBPS\content.opf  -mode opf -version 2.0
Epubcheck Version 3.0-RC-1

WARNING: D:/Temp/Test_9781118388341/OEBPS/content.opf/: File is validated as a s
ingle file of type opf and version 2.0 ! Only a subset of the available tests is
 run
ERROR: D:/Temp/Test_9781118388341/OEBPS/content.opf/D:/Temp/Test_9781118388341/O
EBPS/content.opf(37,64): File listed in reference element in guide was not decla
red in OPF manifest: xhtml/cover.xml
ERROR: D:/Temp/Test_9781118388341/OEBPS/content.opf/D:/Temp/Test_9781118388341/O
EBPS/content.opf(38,69): File listed in reference element in guide was not decla
red in OPF manifest: xhtml/contents.html
Check finished with warnings or errors
```
## checked opf file:

<manifest>
    <item id="Html-cover-page" href="xhtml/cover.xml" media-type="application/xhtml+xml"/>
    <item id="Cover-image" href="images/...jpg" media-type="image/jpeg"/>
    <item id="Contents" href="xhtml/contents.html" media-type="application/xhtml+xml"/>
    ...
</manifest>
<spine toc="ncx">
    <itemref idref="Html-cover-page" linear="no"/>
    <itemref idref="Contents" linear="yes"/>
    ...
</spine>
<guide>
    <reference type="cover" title="Cover" href="xhtml/cover.xml"/>
    <reference type="toc" title="Contents" href="xhtml/contents.html"/>
## </guide>

_Original issue: http://code.google.com/p/epubcheck/issues/detail?id=216_
